packages/broker/test/integration/plugins/operator/InspectRandomNodeHelper.test.ts

```diff
@@ -1,126 +1,84 @@
-import { Contract } from '@ethersproject/contracts'
-import { JsonRpcProvider, Provider } from '@ethersproject/providers'
-import { parseEther } from '@ethersproject/units'
-import { Wallet } from '@ethersproject/wallet'
-import { config } from '@streamr/config'
-import type { TestToken } from '@streamr/network-contracts'
-import { tokenABI } from '@streamr/network-contracts'
+import { fetchPrivateKeyWithGas } from '@streamr/test-utils'
 import { Logger, TheGraphClient, toEthereumAddress, wait, waitForCondition } from '@streamr/utils'
 import fetch from 'node-fetch'
 import { InspectRandomNodeHelper } from '../../../../src/plugins/operator/InspectRandomNodeHelper'
 import { createClient, createTestStream } from '../../../utils'
-import { deploySponsorship } from './deploySponsorshipContract'
-import { setupOperatorContract } from './setupOperatorContract'
-
-const theGraphUrl = `http://${process.env.STREAMR_DOCKER_DEV_HOST ?? '10.200.10.1'}:8800/subgraphs/name/streamr-dev/network-subgraphs`
-
-const logger = new Logger(module)
-const chainConfig = config.dev2
-const STREAM_CREATION_KEY = '0xb1abdb742d3924a45b0a54f780f0f21b9d9283b231a0a0b35ce5e455fa5375e7'
+import {
+    THE_GRAPH_URL,
+    delegate,
+    deploySponsorshipContract,
+    generateWalletWithGasAndTokens,
+    setupOperatorContract,
+    sponsor,
+    stake
+} from './contractUtils'
 
 jest.setTimeout(600 * 1000)
```

Comment:
is this still needed? What's the best practice with timeouts?

Reply:
Would be better to use pattern
```
const TIMEOUT = 600 * 1000

describe('foo', () => {
    beforeEach(() => {
    }, TIMEOUT)
    
    it('bar', () => {
    }, TIMEOUT)
})
```
and so forth.

Reply:
Maybe not, should remove unnecessary waits and decrease timeouts in a separate PR.